DOCSP-27634 replace MongoDB CLI from the Launch Manage page (#104)
mongodb/docs-landing: source/launch-manage.txt

      Learn how to deploy a fully-managed cloud database and get started with a
      free-tier cluster.

   .. card::
      :headline: Atlas CLI
      :url: https://www.mongodb.com/docs/atlas/cli/stable/
      :icon: /images/icons/mcli.svg
      :icon-alt: MongoDB CLI LogoMark

>       Use the command line to interact with or automate management tasks for
>       databases in the Atlas cluster.

Reviewer: suggestion: you can do more with the cli than manage databases. If you need for the text to flow onto another line to make the layout work consider adding `MongoDB` before `Atlas`.
```suggestion
      Use the command line to perform or automate management tasks in Atlas.
```
